src/domUpdates.js

```diff
@@ -160,8 +161,9 @@ const pageLoadRenders = () => {
 };
 
 const showRecipe = (recipeCard) => {
-  const recipeCardName = recipeCard.closest(".individual-recipe").querySelector("h2");
-  const thisRecipe = recipeData.find(recipe => recipe.name === recipeCardName);
+  const recipeCardId = recipeCard.closest("article").id;
```

Comment:
This is a well thought of code. 
Could I just request you to make it 
const recipeCardId = recipeCard.closest("article")?.id;

This would avoid any cases where the event target is not a child of an article.

Reply:
also `ID` should be capitalized in the variable name

Reply:
Done for both! @mbenfowler @taranveersingh93